src/FpdiProtection.php

```diff
@@ -193,26 +193,45 @@ class FpdiProtection extends \setasign\Fpdi\Fpdi
      */
     protected $fileIdentifier;
 
+    /**
+     * @var bool
+     */
+    protected $useArcfourFallback;
+
     /**
      * FpdiProtection constructor.
      *
      * @param string $orientation
      * @param string $unit
      * @param string $size
+     * @param bool $useArcfourFallback
      */
-    public function __construct($orientation = 'P', $unit = 'mm', $size = 'A4')
+    public function __construct($orientation = 'P', $unit = 'mm', $size = 'A4', $useArcfourFallback = false)
     {
         parent::__construct($orientation, $unit, $size);
 
         $randomBytes = function_exists('random_bytes') ? \random_bytes(32) : \mt_rand();
         $this->fileIdentifier = md5(__FILE__ . PHP_SAPI . PHP_VERSION . $randomBytes, true);
+        $this->useArcfourFallback = $useArcfourFallback;
+
+        if($useArcfourFallback) return;
```

Comment:
Add a space `if ($useArcfourFallback)` and use brackets around the `return;` statement.